app/src/components/app.jsx

```diff
@@ -29,4 +27,12 @@ class App extends Component {
 
 }
 
+App.propTypes = {
+  loading: React.PropTypes.bool,
+  children: React.PropTypes.array,
```

Comment:
When running this is actually an object, throwing an error. I expect this to happen elsewhere, this code needs to be run first.